examples/denoise_exr/src/main.rs

```diff
@@ -106,55 +108,54 @@ fn main() {
         .and_then(|d| d.deserialize())
         .unwrap_or_else(|e| e.exit());
 
-    let color = load_exr(&args.flag_c);
-    let mut filter_output = vec![0f32; color.img.len()];
+    let mut color = load_exr(&args.flag_c);
 
     let device = oidn::Device::new();
-    let mut filter = oidn::RayTracing::new(&device);
-    filter
-        .set_srgb(false)
-        .set_hdr(true)
-        .set_img_dims(color.width, color.height);
-
-    if let Some(albedo_exr) = args.flag_a {
-        let albedo = load_exr(&albedo_exr);
-
-        if let Some(normal_exr) = args.flag_n {
-            let normal = load_exr(&normal_exr);
-            filter
-                .execute_with_albedo_normal(
-                    &color.img[..],
-                    &albedo.img[..],
-                    &normal.img[..],
-                    &mut filter_output[..],
-                )
-                .expect("Invalid input image dimensions?");
+
+    #[allow(unused_assignments)]
+    let mut albedo = EXRData::new(0, 0);
```

Comment:
For this and `normal` I think we could make them `let mut albedo: EXRData` ?

Reply:
Yes, I didn't know you can forward declare like this in Rust! Neat. Fixed.